src/DatePickerRange/DatePickerRange.component.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+import { DateRangePicker, DateRangePickerShape } from 'react-dates';
+import uuid from 'uuid';
+import { Moment } from 'moment';
+
+type momentDate = Moment | null;
+type pickerSize = 'sm' | 'md' | 'lg';
+type pickerStyle =
+  | 'default'
+  | 'primary'
+  | 'secondary'
+  | 'success'
+  | 'warning'
+  | 'danger';
+type focusInputs = FocusInputs.START_DATE | FocusInputs.END_DATE | null;
+enum FocusInputs {
+  START_DATE = 'startDate',
+  END_DATE = 'endDate',
+}
+type dateChangeEvent = { startDate: momentDate; endDate: momentDate };
+
+export interface Props extends DateRangePickerShape {
```

Comment:
The propType column is showing `unknown` for all of these types in the storybook documentation. Do you know why?

Reply:
I'm not sure what the reason for that behavior. Researching why is it happens.

Reply:
It happenes if import will be this `import React from 'react'` rather than `import * as React from 'react'`